packages/fe-mockserver-core/test/unit/request/odataRequest.test.ts

```diff
@@ -210,7 +210,7 @@ describe('OData Request', () => {
         let myRequest = new ODataRequest(
             {
                 method: 'GET',
-                url: '/Countries?$expand=Value1,Value2,,Value3,Value4/Value5'
+                url: '/Countries?$expand=Value1,Value2,,Value3,Value4/Value5,Value4/Value6,Value4/Value7,Value4/Value6/Value8'
```

Comment:
```
'/Countries?$expand=Value1,Value2,,Value3,Value4/Value5
                                 ^^ 
```

Anyway, was there before, so it can probably be ignored.

Reply:
yes i thought it was expected :)